service/src/main/java/com/epam/esm/service/utils/CertificateSortParameterValidator.java

```diff
@@ -0,0 +1,29 @@
+package com.epam.esm.service.utils;
+
+import com.epam.esm.model.parameters.CertificateSortType;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class CertificateSortParameterValidator {
+    private static final String PARAMETER_SORT = "sort";
+    private static final char MINUS = '-';
+
+    public static boolean isValidParams(Map<String, String> groupParameters) {
+        Map<String, String> groupSortParameters = groupParameters.entrySet().stream().
```

Comment:
code duplication for getting this `groupSortParams`

Reply:
Will be fixed in the next PR